js/README.md

```diff
@@ -0,0 +1,73 @@
+# Solido JS SDK
+
+Typescript SDK to facilitate interacting with the Solido program in JS/TS applications
+
+## Installation
+
+### Yarn
+
+```
+$ yarn add @chorusone/solido.js
+```
+
+### npm
+
+```
+$ npm install @chorusone/solido.js
+```
+
+## Development
+
+### Install packages
+```
+$ npm install
+or
+$ yarn
+```
+
+### Run TS watch script
+```
+$ npm run watch:ts
+```
+
+### Run tests watch script
+```
+$ npm run watch:test
+```
+
+### Build and Release
+TODO
+
+
+## Usage
+
+### Javascript
+```js
+const SolidoJS = require("@chorusone/solido.js");
```

Comment:
Is it common to use the `JS` suffix even in names in Javascript?

Reply:
Not really, but I just used it in the example here